mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
Yes, I meant `lib/mono/msbuild/15.0/bin/Roslyn -> lib/mono/4.5/`. You can symlink specific files only but I see the duplicated list more like complication than benefit